WcaOnRails/app/models/registration.rb

```diff
@@ -253,6 +253,16 @@ def self.wcif_json_schema
     end
   end
 
+  validate :forcing_competitors_to_add_comment
+  private def forcing_competitors_to_add_comment
+    if !competition.force_comment_in_registration
+      return
+    end
+    if comments.length == 0
```

Comment:
Might be worth trimming/stripping the comment to make sure it's not just whitespaces? `comments.strip.length == 0`

Reply:
Good idea, done.